SU2_CFD/src/solver_direct_elasticity.cpp

```diff
@@ -997,6 +1001,9 @@ void CFEASolver::Set_ElementProperties(CGeometry *geometry, CConfig *config) {
     if (rank == MASTER_NODE){
       cout << "There is no element-based properties file." << endl;
       cout << "The structural domain has uniform properties." << endl;
+      
+      if (topology_mode)
+        SU2_MPI::Error("Topology mode requires an element-based properties file.",CURRENT_FUNCTION);
```

Comment:
Are the property files in ASCII? If they are becoming a bottleneck, you can switch to binary and use MPI IO like we do for the restarts. I can help you with this, and there are some examples you can follow from the restart writer/readers.